### PR TITLE
Bug/1163 implement open sans as font

### PR DESCRIPTION
**Pull request by peggimann**
- write fontface which supports different font weights

### CODE REVIEW

File: frontend/src/app/components/application-banner/application-banner.component.scss [hunk 1]
```diff
@@ -11,3 +11,7 @@ p.mb-2 {
 .filters {
   flex: 1;
 }
+
+.expansion-panel-title {
+  margin-top: 0.2rem;
```
Comment (kcinay055679) [inline frontend/src/app/components/application-banner/application-banner.component.scss:16]: look for alternative

**Review by kcinay055679 — COMMENTED** [review]
create fontfaces for every available font

File: frontend/src/app/components/application-banner/application-banner.component.scss (hunk 1 above)
Comment (ManuelMoeri) [reply frontend/src/app/components/application-banner/application-banner.component.scss:16]: could not find a good alternative.

File: frontend/src/style/styles.scss [hunk 2]
```diff
@@ -10,8 +10,8 @@ body.okr {
   margin: 0;
   padding: 0;
   background-color: $overview-bg;
-  font-family: Roboto, sans-serif;
-  --bs-body-font-family: Roboto, "Helvetica Neue", sans-serif;
+  font-family: OpenSans, sans-serif !important;
```
Comment (peggimann) [inline frontend/src/style/styles.scss:13]: can be removed.

**Pull request merged**